### PR TITLE
feat: Include CNVpytor plots into the CNV report

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

Reformats the generation of a CNV PDF report and includes the CNVpytor plots 🧑‍🎨 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by khurrammaqbool — APPROVED** [review]
💯 
Just a minor comment. We should use *.smk or *.py for pre-rule processing unless inevitable.

File: BALSAMIC/snakemake_rules/quality_control/report.rule [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ascat_plots = expand(
+    vcf_dir + "CNV.somatic." + config["analysis"]["case_id"] + ".ascat." + "{output_suffix}" + ".png",
+    output_suffix=["ascatprofile", "rawprofile", "ASPCF", "tumor", "germline", "sunrise"]
+)
+cnvpytor_plots = expand(
+    vcf_dir + "CNV.somatic." + config["analysis"]["case_id"] + ".cnvpytor." + "{output_suffix}" + ".png",
+    output_suffix=["circular", "scatter"]
+)
+sample_statistics = ascat_statistics if config['analysis']['analysis_type'] == "paired" else []
+sample_plots = ascat_plots if config['analysis']['analysis_type'] == "paired" else cnvpytor_plots
```
[the remark this answers is not in the document]
Comment (ivadym) [reply BALSAMIC/snakemake_rules/quality_control/report.rule:14]: Done 👍

**Comment by ivadym** [comment]
### Tested cnv_report rule ✅ :

#### TO-WGS report:
[CNV.somatic.civilsole.report.pdf](https://github.com/Clinical-Genomics/BALSAMIC/files/10095285/CNV.somatic.civilsole.report.pdf)

#### TN-WGS report:
[CNV.somatic.fleetjay.report.pdf](https://github.com/Clinical-Genomics/BALSAMIC/files/10095286/CNV.somatic.fleetjay.report.pdf)

**Comment by ivadym** [comment]
After updating the tags (https://github.com/Clinical-Genomics/hermes/pull/67):

Screen Shot 2022-11-26 at 15 23 20

Screen Shot 2022-11-26 at 15 25 33

**Pull request merged**